Tools/LyTestTools/ly_test_tools/o3de/asset_processor.py

```diff
@@ -654,7 +655,9 @@ def set_failure_log_folder(self, log_root):
     def check_copy_logs(self):
         if self._temp_asset_root and self._failed_log_root:
             source_path = os.path.join(self._temp_asset_root, "logs")
-            dest_path = os.path.join(self._failed_log_root, f"{self._function_name + '.' if self._function_name else ''}{int(round(time.time() * 1000))}")
+            dest_path = os.path.join(self._failed_log_root,
+                                     f"{self._function_name + '.' if self._function_name else ''}"
+                                     f"{int(round(time.time() * 1000))}")
```

Comment:
The line break between 659 and 660 makes this less readable to me, prompting me think about how the precedence will work between if/else and string concatenation.

Recommend resolving the if/else outside of the call to join and storing it in a variable with a name similar to `failed_log_folder`

Reply:
Yeah the splitting of if/else string concats did not play nice. fixed to suggestion